profile/module_test.go

```diff
@@ -1,11 +1,7 @@
 package profile
 
-import (
-	"testing"
-
-	"github.com/safing/portmaster/core/pmtesting"
-)
-
+/*
 func TestMain(m *testing.M) {
 	pmtesting.TestMain(m, module)
 }
+*/
```

Comment:
Maybe we should remove the whole file instead so we are not "tricked" by "everything's fine because there's a test-file".